### PR TITLE
Metflix : Added header

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
Hello @Zilola-Nazarova ,
In this branch, I did the following
-Added the header section
-Did minimal styling of the header section to make it look presentable
-Added Hover effect on the options
From this, the Single Page applications will be created.

The logo is not displaying, yet not sure why.
This is the display of the header section
image

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,6 +7,24 @@
   <title>Document</title>
 <script defer src="main.js"></script></head>
 <body>
-  <h1>Hi Arnold!</h1>
+  <header>
+    <img src="assets/metflixlogo2.png" alt="Metflix" class="logo">
```
Comment (Zilola-Nazarova) [inline dist/index.html:11]: Try to use `require` like this:
```
<img src=<%=require("./assets/metflixlogo2.png")%> alt="Metflix" class="logo">
```

_PS: Honestly, I don't know why it should work like this. But I had similar issues in previous projects and found this [article](https://stackoverflow.com/questions/60302749/how-to-simply-load-images-with-src-attribute-with-webpack-4) on StackOverflow. And they suggested this approach._

**Review by Zilola-Nazarova — CHANGES_REQUESTED** [review]
Hi @arnoldnekemiah 
The header looks good and I'll approve it 💯 
I just had a look and tried to understand why the logo isn't displaying.
It seems like you need to load the assets in index.html in a different way.
Check the comment below.
Let me know if it helped 💁🏻

File: dist/index.html (hunk 1 above)
Comment (arnoldnekemiah) [reply dist/index.html:11]: It has worked, thanks

**Review by Zilola-Nazarova — APPROVED** [review]
bravocado
Keep up the geat work! 👏🏻

**Pull request merged**